docs/UserGuide.md

```diff
@@ -187,18 +188,18 @@ Examples:
 
 ### Deleting a customer : `deletecustomer`
 
-Deletes the specified customer from the address book.
+Deletes the specified customer contact and order list from the address book.
 
 Command aliases: `deletec`<br>
 Format: `deletecustomer CUSTOMER_INDEX`
 Examples:
-* `listcustomer` followed by `deletecustomer 2` deletes the 2nd customer in the address book.
-* `findcustomer Betsy` followed by `deletec 1` deletes the 1st customer in the results of the `find` command.
+* `listcustomer` followed by `deletecustomer 2` deletes the 2nd customer contact and order list in the address book.
```

Comment:
```suggestion
* `listcustomer` followed by `deletecustomer 2` deletes the 2nd customer contact in the address book, along with their corresponding order list.
```

Reply:
Not sure if editing this line is important since at this point deleting order list should be implied